portfolio/src/main/client/src/components/pages/index.ts

```diff
@@ -0,0 +1,31 @@
+import {CookingSection, DishInfo} from '@src/components/sections/cooking';
+import {htmlElement} from '@src/util/html';
+
+export const IndexPage = (): HTMLElement => {
+  const dishContext = require.context('@res/text/dish', false);
+  const dishInfos = dishContext.keys().map((key) => dishContext(key) as DishInfo);
```

Comment:
Can you explain what's going on here? Potentially with a comment. I'm not sure I follow where `dishContext` comes from or how you're converting your markdown to html.

Reply:
Added an explanation.